clients/spark/core/src/test/scala/io/treeverse/clients/GarbageCollectorSpec.scala

```diff
@@ -145,6 +145,51 @@ class GarbageCollectorSpec extends AnyFunSpec with Matchers with SparkSessionSet
         })
       }
 
+      it(
+        "should report all elements in expired ranges that are not imported (i.e. in the same namespace of the repo) and keep others"
+      ) {
+        withSparkSession(spark => {
+          import spark.implicits._
+          val storageNamespace = "s3://repo-ns"
+          val importedDataS3Path = "s3://bucket/of/imported/data"
+          val repoName = "repo-name"
+          val rangeGetter = new ARangeGetter(repoName,
+                                             null,
+                                             Map(
+                                               "aaa" -> Seq("a1",
+                                                            s"$importedDataS3Path/a2",
+                                                            s"$storageNamespace/a3"
```

Comment:
If you define these as functions, you might instead write
```suggestion
                                                            importedDataS3Path("a2"),
                                                            storageNamespace("a3")
```
Might be clearer?